circle.eslint.json

```diff
@@ -14,6 +14,7 @@
         "aliases": ["applications", "site", "vet360"]
       }
     ],
-    "react/prefer-stateless-function": 2
+    "react/prefer-stateless-function": 2,
+    "sonarjs/cognitive-complexity": [1, 50]
```

Comment:
Due to this being a warning, this won't show up in our logs.  It may be good if `lint:circle` reports on warnings as well as errors:

```diff
diff --git a/package.json b/package.json
index 9c1095887..bdce837d9 100644
--- a/package.json
+++ b/package.json
@@ -18,7 +18,7 @@
     "lint": "npm run lint:js && npm run lint:sass",
     "lint:changed": "npm run lint:js:changed && npm run lint:sass",
     "lint:js": "eslint --quiet --ext .js --ext .jsx .",
-    "lint:circle": "eslint --quiet --ext .js --ext .jsx -c circle.eslint.json .",
+    "lint:circle": "eslint --ext .js --ext .jsx -c circle.eslint.json .",
     "lint:js:fix": "eslint --fix --quiet --ext .js --ext .jsx .",
     "lint:js:changed": "LIST=`git diff-index --name-only --cached HEAD | grep \"\\.js[x]\\{0,1\\}$\"`; if [ \"$LIST\" ]; then eslint --quiet $LIST; fi",
     "lint:js:changed:fix": "LIST=`git diff-index --name-only HEAD | grep \"\\.js[x]\\{0,1\\}$\"`; if [ \"$LIST\" ]; then eslint --fix --quiet $LIST; fi",
```

Reply:
That's a great idea!